source/using-api-interface.rst

```diff
@@ -258,11 +257,11 @@ If there is no output, the message was sent to MQTT broker.
 **Also you can send a configuration message from dojot to the device to change some of its attributes.
 The target attribute must be of type “actuator”.**
 
-To simulate receiving the message on a device, we can use the mosquitto_sub:
+To simulate receiving the message on a device, we can use ``mosquitto_sub``:
 
 .. code-block:: bash
 
-  mosquitto_sub -h localhost -p 1883  -i admin:0998 -t /admin/0998/config
+  mosquitto_sub -h localhost -p 1883 -u admin:0998 -t admin:0998/config -q 1
```

Comment:
Maybe you can explain why to use qos=1, a short explanation.  Or maybe here isn't the right place.